### DISCUSSION TITLE
Unknown MySQL server host

### DISCUSSION THREAD

**Discussion by vijayavanisabbiti** [learndevopsonline/learndevopsonline#1694] [Q&A] [answered]
Getting MySQL error

<img width="740" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/150422689/45afc4fb-a4b7-4dcf-957c-19622a425d52">

I suspect my-sql route53 record is not created with terraform code.

<img width="636" alt="image" src="https://github.com/learndevopsonline/learndevopsonline/assets/150422689/55f097a6-22cf-4ef6-ae91-e805f33c7b1e">

If yes, how can this be created? please help.

**Comment by sreedharm07** [comment, marked as answer]
you are giving ami machines password for mysql server load schema
![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/7aa91c3f-a919-45a6-a249-9bca20bd3db7)

**Reply by vijayavanisabbiti** [reply]
Thank you Sreedhar. This is solved.
